README_ENG.md

```diff
@@ -0,0 +1,103 @@
+[한국어](https://github.com/yf-dev/OpenCaptionsOverlay)
+
+# Open Captions Overlay
+
+![Sample](./sample.gif)
+
+`Open Captions Overlay` is a service to display subtitles on broadcast programs such as `OBS` and `XSplit`.
+
+If you use this service, the voice spoken through the microphone can be displayed as subtitles in real time on the broadcast screen.
+
+This project was inspired by [Closed Captions for Streams](https://www.twitch.tv/ext/xxwoffr2lnpxrgpq228mawvdgxetip).
+
+[Demo Video](https://youtu.be/CAIVO6aMgs4)
+
+## Usage
+
+### 1. Setting the speech recognition
+
+1. Access [Voice Recognition Site] (https://cc-overlay.update.sh/recognition) using __"latest version of Chrome"__.
+2. Log in to the `Voice Recognition Site` with the Twitch account you want to broadcast.
+3. Click the `Start Recognition` button and allow the microphone permission.
```

Comment:
English text does not exist currently.

Reply:
Yeah

Reply:
Although the website does not provide English UI yet, I'll merge this PR for now.